src/app/globals.css

```diff
@@ -47,6 +47,10 @@
   /* Profile */
   --profile-x-gap: 20px;
   --profile-y-gap: 15px;
+  --toggle-slider-radius: 18px;
+  --slider-thumb-radius: 18px;
+  --slider-thumb-border-radius: 10px;
+  --slider-thumb-box-shadow: 0 1px 2px 1px #00000080;
```

Comment:
Take this to local scope. If you want to use variables, make one of the root/container's css rule set hold that variable.